core/process/setsid_spec.rb

```diff
@@ -21,13 +21,17 @@
       write.close
       read2.close
       pgid_child = read.gets
+      pgid_child = Integer(pgid_child)
```

Comment:
Do you mean moving "pgid_child = Integer(pgid_child)" into the "platform_is_not :aix" block? But pgid_child is used not only at line 30 but also at line 35.